packages/front-end/components/Features/FeatureUsageGraph.tsx

```diff
@@ -0,0 +1,163 @@
+import { Bar } from "@visx/shape";
+import { scaleBand, scaleLinear } from "@visx/scale";
+import { ParentSizeModern } from "@visx/responsive";
+import { Group } from "@visx/group";
+import React, { createContext, ReactNode, useContext, useEffect } from "react";
+import {
+  FeatureUsageData,
+  FeatureUsageTimeSeries,
+} from "back-end/types/feature";
+import { FeatureUsageLookback } from "back-end/src/types/Integration";
+import useApi from "@/hooks/useApi";
+import { useLocalStorage } from "@/hooks/useLocalStorage";
+import { growthbook } from "@/services/utils";
+
+const formatter = Intl.NumberFormat("en-US", {
+  notation: "compact",
+  maximumFractionDigits: 1,
+});
+
+const featureUsageContext = createContext<{
+  lookback: FeatureUsageLookback;
+  setLookback: (lookback: FeatureUsageLookback) => void;
+  featureUsage: FeatureUsageData | undefined;
+  mutateFeatureUsage: () => void;
+}>({
+  lookback: "15minute",
+  setLookback: () => {},
+  featureUsage: undefined,
+  mutateFeatureUsage: () => {},
+});
+
+export function FeatureUsageProvider({
+  featureId,
+  children,
+}: {
+  featureId: string;
+  children: ReactNode;
+}) {
+  const [lookback, setLookback] = useLocalStorage<FeatureUsageLookback>(
+    "featureUsageLookback",
+    "15minute"
+  );
+
+  const useFeatureUsage = growthbook.isOn("feature-usage");
+
+  const { data: featureUsage, mutate: mutateFeatureUsage } = useApi<{
+    usage: FeatureUsageData;
+  }>(`/feature/${featureId}/usage?lookback=${lookback}`, {
+    shouldRun: () => useFeatureUsage,
```

Comment:
We should also only run this if the org has a valid growthbook data source.  Otherwise, the API calls are just going to 4xx immediately.